README.md

```diff
@@ -6,6 +6,16 @@
 
 An extremely fast Python type checker and language server, written in Rust.
 
+## Highlights
+
+- 10x - 60x faster than mypy and Pyright
+- Support for features not allowed in other Python type checkers, e.g., redeclarations
```

Comment:
I'm not that into this one, probably another way to frame it or other features to highlight, e.g., the gradual guarantee